### DISCUSSION TITLE
Encoder x264 recording blurred

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [obsproject/obs-studio#5926] [Development Questions]
### Operating System Info

Windows 10

### Expected Behavior

Use this default value to solve the Game Recording BlurRed
> * [ ] `Output Mode` `Advancced` Default `Bitrate 5000kbps` Default 
> * [ ] `Video` Downscale Filter `Lanczos (Sparpened Scaling, 36 Samples)`

### Current Behavior

`Output Mode` `Advancced` Default `Bitrate 2500kbps` Default 
`Video` Downscale Filter `Bicubic (Sparpened Scaling, 16 Samples)`

**Comment by Xorshasia** [comment]
Might want to drop by the Official OBS Discord server as this seems more a support issue than any sort of bug.

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for assistance.

Thank you!
